src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs

```diff
@@ -15,7 +15,7 @@
 (react/defc Page
   {:render
    (fn [{:keys [state refs this]}]
-     [:div {:style {:textAlign "center"}}
+     [:div {:style {:textAlign "center" :width "50vw"}}
```

Comment:
http://caniuse.com/#feat=viewport-units known issues says that Chrome will not print elements using "vw" widths. Can you test this? And ... do we care? (probably don't care for this dialog; raising the issue for future dialogs)